src/index.html

```diff
@@ -99,10 +320,52 @@ <h2 class="quiz-card__heading">Question</h2>
         </ul>
       </section>
     </main>
-
+    <!-- create -->
+    <main class="page-create hidden">
+      <form method="post" class="create-form">
+        <div class="create-form__input-group">
+          <label for="question" class="create-form__heading">Question</label>
+          <textarea
+            name="question"
+            id="question"
+            cols="30"
+            rows="5"
+            placeholder="Add your question here ..."
+            required
+          ></textarea>
+        </div>
+        <div class="create-form__input-group">
+          <label for="answer" class="create-form__heading">Answer</label>
+          <textarea
+            name="answer"
+            id="answer"
+            cols="30"
+            rows="8"
+            placeholder="Add your answer here ..."
+            required
+          ></textarea>
+        </div>
```

Comment:
Another way also, to save yourself a `div` element 😏 You could nest the textarea within the label and then give the styling you give the div to the label. Should work the same :)
```suggestion
       <label class="create-form__label">Answer
          <textarea
            name="answer"
            cols="30"
            rows="8"
            placeholder="Add your answer here ..."
            required
          ></textarea>
        </label>
```